.prettierrc.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  arrowParens: "avoid",
```

Comment:
prettier should automatically detect the eslint and use eslint configuration

Reply:
removed the config